api/handler/v2/tx.go

```diff
@@ -73,3 +84,143 @@ func (h *Handler) BroadcastTx(ctx *fasthttp.RequestCtx) {
 	}
 	sendSuccessfulJSONResponse(ctx, &response)
 }
+
+func ValidateBroadcastTxParams(params *BroadcastTxParams) error {
+	// TODO: validate network by checking if it's in the list of available networks
+	if params.Network == "" {
+		return fmt.Errorf("network cannot be empty")
+	}
+	if len(params.TxBytes) == 0 {
+		return fmt.Errorf("tx_bytes cannot be empty")
+	}
+	return nil
+}
+
+// BroadcastTxParams represents the parameters for the POST /v2/tx/broadcast endpoint.
+type BroadcastAminoTxParams struct {
+	// which network should the transaction be broadcasted to
+	Network   string                `json:"network"`
+	Signed    legacytx.StdSignDoc   `json:"signed"`
+	Signature legacytx.StdSignature `json:"signature"` //nolint:staticcheck
+}
+
+type BroadcastAminoTxResponse struct {
+	Code   uint32 `json:"code"`
+	TxHash string `json:"tx_hash"`
+	RawLog string `json:"raw_log"`
+}
```

Comment:
types need to be defined on top of the file

Reply:
I am writing types align with the function / endpoint for which they are required in this handler files. But happy to change it if you would rather have them at the top of file.

Reply:
yes, please change as it's part of our style guide